src/main/java/org/spongepowered/api/event/entity/living/LivingEvent.java

```diff
@@ -37,8 +38,5 @@
      *
      * @return The {@link Living} involved
      */
-    Living getLiving();
-
-    @Override
-    Living getEntity();
+    Entity getEntity();
```

Comment:
Java doesnt allow use of anything but entity here, thus why it was used. Much would have preferred it to be living.